setup.py

```diff
@@ -23,9 +23,9 @@ def write_version_py(filename=os.path.join('pytripgui', 'VERSION')):
 
 # install_requires is list of dependencies needed by pip when running `pip install`
 install_requires = [
-    'pytrip98~=3.7',
+    "matplotlib==3.4.3 ; python_version>='3.7'",
```

Comment:
What about this one ?

```suggestion
    "matplotlib<=3.4.3 ; python_version>='3.7'",
```

Reply:
Same for requirements.txt

Reply:
Then we should also remove the python_version condition, which was included as a workaround to python 3.5 and 3.6 only being supported by matplotlib versions up to version 3.3.

Reply:
But imagine user has matplotlib 3.4.1 and python 3.8.
Do we need to enforce upgrade from 3.4.1 to 3.4.3 ?

Reply:
In turn, this might introduce a problem with someone having a system-wide installation of a much older matplotlib version than 3.4.3. That version would also satisfy this requirement but might not support all of the features we use either.

As stated in the comment in requirements.txt, this line will be removed once PyTRiPGUI supports the newest version of Matplotlib which is installed by PyTRiP.

Reply:
ok, then I agree